### PR TITLE
Update Corel.WinZip to 28.0.15640

### PR DESCRIPTION
**Pull request by FaserF**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
This is another try of the old PR https://github.com/microsoft/winget-pkgs/pull/138830 where only the PackageDependencies failed.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151794)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-05-01 05:27:48.827 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFD54FFC11F: (caller: 00007FFD54D88D6F) LogHr(1) tid(a34) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD54F02B08: (caller: 00007FFD54FABC20) Exception(1) tid(a34) 8A15005E ] 
> 2024-05-01 05:27:48.827 [REPO] Failed to open available source: msstore
> 2024-05-01 05:27:57.087 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFD3E31C11F: (caller: 00007FFD3E0A8D6F) LogHr(1) tid(1454) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFD3E222B08: (caller: 00007FFD3E2CBC20) Exception(1) tid(1454) 8A15005E ] 
> 2024-05-01 05:27:57.087 [REPO] Failed to open available source: msstore
> 2024-05-01 05:29:01.820 [CLI ] ShellExecute installer failed: 1603
> Property(S): ErrorDialog = SetupError
> Property(S): S_ERROR_TITLE = WinZip Install Error
> Property(S): WinZipErrorPageUrl = http://www.winzip.com/wzgate.cgi?lang=EN&url=www.winzip.com/net4update.html
> MSI (s) (68:1C) [05:29:01:661]: Product: WinZip 28.0 -- Installation operation failed.

Automated error analysis suggests 1603 may mean Install successful but system restart may be required.

Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.

(Automated response - build 878.)

**Comment by FaserF** [comment]
Looks top me like a dependency issue and not a package issue

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   2FA967895ADEDA18C1FC3C34C3C46CFC9660695ACFA668CA591E2CE8B0F1DE93
 Expected hash: 729df905fd1086d611042689117d78a0b5ad7a7b2d29bba291eb36d9233c69fb
 
 5/7/2024 12:17:26 AM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 880.)

File: manifests/c/Corel/WinZip/28.0.15640/Corel.WinZip.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 28.0.15640
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.winzip.com/winzip280-64.msi
+  InstallerSha256: 729DF905FD1086D611042689117D78A0B5AD7A7B2D29BBA291EB36D9233C69FB
```
Comment (stephengillie) [inline manifests/c/Corel/WinZip/28.0.15640/Corel.WinZip.installer.yaml:14]:
```suggestion
  InstallerSha256: 2FA967895ADEDA18C1FC3C34C3C46CFC9660695ACFA668CA591E2CE8B0F1DE93
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-05-12 22:21:25.310 [REPO] Failed to open available source: msstore
> 2024-05-12 22:21:33.819 [REPO] Failed to open available source: msstore
> 2024-05-12 22:22:47.783 [CLI ] ShellExecute installer failed: 1603
> Property(S): ErrorDialog = SetupError
> Property(S): S_ERROR_TITLE = WinZip Install Error
> Property(S): WinZipErrorPageUrl = http://www.winzip.com/wzgate.cgi?lang=EN&url=www.winzip.com/net4update.html
> MSI (s) (A0:40) [22:22:47:525]: Product: WinZip 28.0 -- Installation operation failed.

Automated error analysis suggests 1603 may mean Install successful but system restart may be required.

Automated error analysis suggests -1978335226 may mean Install successful but system restart may be required.

(Automated response - build 880.)

**Comment by FaserF** [comment]
This is a bug with Winget itself, right?
This package does not need msstore, still this error occurs:

Failed to open available source: msstore

**Comment by stephengillie** [comment]
> This is a bug with Winget itself, right? This package does not need msstore, still this error occurs:
> 
> Failed to open available source: msstore

I think this might be looking up the dependency. I'm not sure. This specific line seems to occur frequently without any corellation to a specific error, so I think I'll add it to the "filter out" list. 
- There was a [suggestion](https://github.com/microsoft/winget-pkgs/pull/148226#issuecomment-2045536064) to remove this source from validation, "to reduce the log load here."

For the `exit code` and `Shellexeute installer` codes, should `1603` be added as an `InstallerSuccessCode` or similar?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**